### ISSUE TITLE
Update to PureScript 0.10.x?

### ISSUE THREAD

**Issue by FranklinChen** [parsonsmatt/purs-architecture-tutorial#5]
I realize one sometimes has to let blog posts stay pinned to a point in history and not perpetually update them, but do you plan to update the code to more recent versions of PureScript and/or Halogen?

**Comment by parsonsmatt** [comment]
I would! I hear that Halogen is in the middle of a big update, so I'll try to stay on top of that and do an update when it happens.

**Comment by FranklinChen** [comment]
Yeah, I've actually held off on using Halogen because it keeps on changing!

**Comment by FranklinChen** [comment]
Amusingly, PureScript 0.11 caused yet more changes.

**Comment by parsonsmatt** [comment]
At this point I might be waiting for 1.0 :sweat_smile:

**Comment by FranklinChen** [comment]
Yeah. PureScript 0.12 recently radically changed a bunch of things yet again.
